R/pkg/R/DataFrame.R

```diff
@@ -1384,7 +1384,7 @@ setMethod("saveAsTable",
                                     "org.apache.spark.sql.parquet")
             }
             allModes <- c("append", "overwrite", "error", "ignore")
-            if (!(mode %in% allModes)) {
+            if (! (mode %in% allModes)) {
```

Comment:
we also need a nolint here (and to roll back the change to add space ?) (line 1389)